pom.xml

```diff
@@ -421,6 +421,8 @@
                 <exclude>**/.gitattributes</exclude>
```

Comment:
How is the Gradle plugin being released ?
Since it is not a sub-module of the Maven parent pom what steps would be needed to release it with 1.13.0 ?